testproject/Assets/Tests/Runtime/Animation/AnimatorTestHelper.cs

```diff
@@ -29,10 +29,12 @@ private void Awake()
             m_NetworkAnimator = GetComponent<NetworkAnimator>();
         }
 
+#if UNITY_EDITOR
```

Comment:
I understand the `#if` in `NetworkAnimatorTests.cs`. But why do need this one too? Couldn't we leave this expose, always, but not called outside the editor ?

Reply:
We could, but why keep an internal method for runtime builds that will never be used during runtime and is not part of the SDK?